index.html

```diff
@@ -0,0 +1,188 @@
+                <ul class="program">
+                    <li class="list">
+                        <img src="important-img/lecture-icon.png" alt="lecture-icon">
+                        <h3 class="list-header">
+                            Lecture
+                          </h3>
+                          <p class="list-para">
+                            Listen to the speakers from various countries about the messages of sharing and opening.
+                          </p>
+                    </li>
+                    <li class="list">
+                        <img src="important-img/exhibition-icon.png" alt="exhibition-icon">
+                        <h3 class="list-header">
+                            CC Exhibition
+                          </h3>
+                          <p class="list-para">
+                            Appreciate various creations applying CC license organized by Enercare centre, Toronto.
+                          </p>
+                    </li>
+                    <li class="list">
+                        <img src="important-img/forum-icon.png" alt="forum-icon">
+                        <h3 class="list-header">
+                            Forum
+                          </h3>
+                          <p class="list-para">
+                            Have the time to share your thoughts and opinions with experts for each topic.
+                          </p>
+                    </li>
+                    <li class="list">
+                        <img src="important-img/workshop-icon.png" alt="workshop-icon">
+                        <h3 class="list-header">
+                            Workshop
+                          </h3>
+                          <p class="list-para">
+                            Try creating your own work using open source license rather than just watching at it.
+                          </p>
+                    </li>
+                    <li class="list">
+                        <img src="important-img/ignite-icon.png" alt="ignite-icon">
+                        <h3 class="list-header">
+                            CC Ignite
+                          </h3>
+                          <p class="list-para">
+                            Get opportunities to network with CC affiliates around the world, also after the summit.
+                          </p>
+                    </li>
+                </ul>
+                <p class="parr"><a href="#">SEE THE WHOLE PROGRAM</a>
+                </p>
+                <div class="join">
+                    <h3>
+                      Join CC Global Summits 
+                    </h3>
+                  </div>
+            </section>
+
+            <section id="guests" class="sec3">
+                <h2 class="sec-2-title">
+                    Featured Speakers
+                  </h2>
+                  <hr class="line">
+                  <ul class="speakers">
+                    
+                  </ul>
+            </section>
```

Comment:
Thankyou for the feedback 
but in the curriculum exercise it was mentioned that the more button is optional

Reply:
> Thankyou for the feedback @brainconnect93 but in the curriculum exercise it was mentioned that the more button is optional

Reply:
> * [ ]  Great effort creating your `Feature Speakers` section of the project dynamically, but currently, on the `mobile view` of the project there are some things that you need to work on.
>   
>   * firstly, on the original design the `mobile view` has only `2 feature speakers` and then the `MORE` button which optionally you can implement a function whereby if a user clicks on the `MORE` button the remaining `4 feature speakers` will be displayed.
>   * secondly, on your `mobile view` the `MORE` button is missing, kindly add it

> 
> Your Design 👇
> 
> Mobile	Mobile
> ![rik1](https://user-images.githubusercontent.com/78634841/217526378-2854254b-65c0-4e2f-a26d-c54b8b7b6f1e.png)	![rik2](https://user-images.githubusercontent.com/78634841/217526455-cb9ae7b9-0822-4697-b8f3-5238ebde438f.png)

 
Thankyou for the feedback @brainconnect93
but in the curriculum exercise it was mentioned that the more button is optional 😢 ????

Reply:
@ritikarawat220, the more button functionality is what is made optional, and from my review, I said optionally you can implement the function, that's if you like and if otherwise just put the button there alone without any onClick function